pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -9785,10 +9765,10 @@ def test_print_object_standard(sma_mir, hera_uvh5):
     Check that the 'standard' mode of print_object works.
     """
     check_str = (
-        "   ID     Cat Entry       Type     Az/Lon/RA    El/Lat/Dec  Frame    Epoch \n"
-        "    #          Name                    hours           deg                 \n"
-        "---------------------------------------------------------------------------\n"
-        "    1          3c84   sidereal    3:19:48.16  +41:30:42.11    fk5  J2000.0 \n"
+        "   ID     Cat Entry          Type     Az/Lon/RA    El/Lat/Dec  Frame    Epoch \n"  # noqa
```

Comment:
Is the #noqa needed in tests?

Reply:
It's needed for the linting to pass because the line is too long. I could wrap the line, but it's only slightly too long so I thought it was more readable like this.